llvm/CMakeLists.txt

```diff
@@ -462,6 +462,10 @@ if( LLVM_USE_PERF )
   endif( NOT CMAKE_SYSTEM_NAME MATCHES "Linux" )
 endif( LLVM_USE_PERF )
 
+option(SYCL_BUILD_PI_CUDA
```

Comment:
We ifdef in/out SYCL NVPTX support in the clang Driver based on this variable, which is why we put it in `llvm/CMakeLists.txt` rather than in `llvm/sycl/CMakeLists.txt`

Reply:
The name of the option is confusing in this case. PI has nothing to do with the driver.
Why do we need ifdef out SYCL NVPTS support in the clang driver?

Reply:
We don't need to, but it helps prevent users accidentally compiling kernels to the wrong target for the plugins they have available.

Reply:
Are you able to compile kernels for PTX w/o CUDA toolchain?

Reply:
I so, we could use this for testing SYCL for CUDA compiler using cross-compilation on Intel HW.

Reply:
I believe clang will fail to produce PTX kernels without a CUDA toolchain installed.

Reply:
We could enable support for the SYCL cuda toolchain if the NVPTX target is added, so it has the same requirements as the CUDA support in llvm? I suggest we do that on a separate patch

Reply:
Note, the code added under ifdef is not covered by CI system, so it might be broken.
It seems to me that in order to enable compilation for NVPTX target we need just a few changes. Am I wrong?